arb_os/constants.json

```diff
@@ -0,0 +1,836 @@
+      9
+    ],
+    [
+      "TxResultCode_unknownFailure",
+      255
+    ],
+    [
+      "EVMCallType_call",
+      0
+    ],
+    [
+      "EVMCallType_callcode",
+      1
+    ],
+    [
+      "EVMCallType_delegatecall",
+      2
+    ],
+    [
+      "EVMCallType_staticcall",
+      3
+    ],
+    [
+      "EVMCallType_constructor",
+      4
+    ],
+    [
+      "LogType_txReceipt",
+      0
+    ],
+    [
+      "LogType_blockSummary",
+      1
+    ],
+    [
+      "LogType_send",
+      2
+    ],
+    [
+      "LogType_sendMerkleTree",
+      3
+    ],
+    [
+      "SendType_sendTxToL1",
+      3
+    ],
+    [
+      "AVMSendType_batch",
+      0
+    ],
+    [
+      "InitOption_setChargingParams",
+      2
+    ],
+    [
+      "InitOption_setDefaultAggregator",
+      3
+    ],
+    [
+      "Charging_DefaultL1GasPerL2Tx",
+      3700
+    ],
+    [
+      "Charging_DefaultL1GasPerCalldataByte",
+      16
+    ],
+    [
+      "Charging_DefaultL1GasPerStorage",
+      2000
+    ],
+    [
+      "Charging_DefaultArbGasDivisor",
+      10000
+    ],
+    [
+      "Charging_AssumedBatchCostL1Gas",
+      50000
+    ],
+    [
+      "Charging_GasPoolDepthSeconds",
+      60
+    ],
+    [
+      "Charging_RetryableTxRepriceIntervalSeconds",
+      900
+    ],
+    [
+      "NetFee_defaultRateNumerator",
+      15
+    ],
+    [
+      "NetFee_defaultRateDenominator",
+      100
+    ],
+    [
+      "Sequencer_maxDelaySeconds",
+      983040
+    ],
+    [
+      "Sequencer_deltaSeconds",
+      0
+    ],
+    [
+      "PluggableModuleID_rollupTracker",
+      0
+    ],
+    [
+      "PluggableModuleID_precompile_0x05",
+      1
+    ],
+    [
+      "RetryBuffer_DefaultLifetimeSeconds",
+      604800
+    ],
+    [
+      "EVMWriteL1GasCost",
+      5000
+    ],
+    [
+      "EVMNonZeroBalanceCallStipend",
+      2300
+    ],
+    [
+      "ArbitrumNonZeroBalanceCallStipend",
+      20000
+    ],
+    [
+      "TwoToThe32",
+      4294967296
+    ],
+    [
+      "SecondsPerBlockNumerator",
+      2
+    ],
+    [
+      "SecondsPerBlockDenominator",
+      1
+    ],
+    [
+      "DefaultSpeedLimitPerSecond",
+      100000000
+    ],
+    [
+      "DefaultSecondsPerSend",
+      900
+    ],
+    [
+      "Estimate_L1GasCostPerNode",
+      220000
+    ],
+    [
+      "Estimate_L1GasPrice",
+      100000000000
+    ]
+  ],
+  "hex": [
+    [
+      "Charging_DefaultNetworkFeeRecipient",
+      "0"
+    ],
+    [
+      "Charging_DefaultCongestionFeeRecipient",
+      "0"
+    ],
+    [
+      "SpecialAccount_gasAccountingReserve",
+      "af6cbc19f66dec07f790912226744d744f04b37b666b9343317df33a5114fb96"
+    ],
+    [
+      "BLSSignatureDomainBase",
+      "73a92f91d473214defd5ffa91d036007eb2e6487fffaa551835e988fb24aaa2b"
+    ],
+    [
+      "EVMLogTopicForL2ToL1Send",
+      "5baaa87db386365b5c161be377bc3d8e317e8d98d71a3ca7ed7d555340c8f767"
+    ]
+  ],
+  "contract": [
+    "ArbAggregator",
```

Comment:
Might be best to have these be relative paths so that the format is more extensible in the future